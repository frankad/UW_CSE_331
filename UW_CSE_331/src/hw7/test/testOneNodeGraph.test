# Test a case creating a graph with only one node.

# Load oneNodeGraph.tsv
LoadGraph oneNodeGraph oneNodeGraph.tsv

# Print out the list of nodes in this graph and their out going edges for each node
ListNodes oneNodeGraph
ListChildren oneNodeGraph Eva