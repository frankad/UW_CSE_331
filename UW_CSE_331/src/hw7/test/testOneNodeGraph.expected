# Test a case creating a graph with only one node.

# Load oneNodeGraph.tsv
loaded graph oneNodeGraph

# Print out the list of nodes in this graph and their out going edges for each node
oneNodeGraph contains: Eva
the children of Eva in oneNodeGraph are: